voluncheer/chatroom/admin.py

```diff
@@ -1,3 +1,3 @@
-from django.contrib import admin
+# from django.contrib import admin
```

Comment:
Instead of commenting these out we should remove these files until we need them. There is no sense in having modules that don't do anything.

Here and elsewhere.